sqlglot/dialects/spark.py

```diff
@@ -41,6 +41,17 @@ class Parser(Spark2.Parser):
         }
 
     class Generator(Spark2.Generator):
+        TYPE_MAPPING = {
+            **Spark2.Generator.TYPE_MAPPING,
+            exp.DataType.Type.TINYINT: "TINYINT",
+            exp.DataType.Type.SMALLINT: "SMALLINT",
+            exp.DataType.Type.BIGINT: "BIGINT",
+            exp.DataType.Type.MONEY: "DECIMAL(15, 4)",
+            exp.DataType.Type.SMALLMONEY: "DECIMAL(6, 4)",
+            exp.DataType.Type.VARBINARY: "BINARY",
```

Comment:
This is redundant, `VARBINARY` is already mapped to `"BINARY"` in the Hive dialect, which is a superclass.